check50.py

```diff
@@ -42,8 +43,13 @@ def main():
 
     # check for newer version on PyPi
     pypi = pypijson.get("check50")
-    if pypi and not args.force and StrictVersion(pypi["info"]["version"]) > VERSION:
-        raise RuntimeError("You are running an old version of check50. Run pip install check50 --upgrade, and then run check50 again!")
+    if pypi and not args.no_autoupdate and StrictVersion(pypi["info"]["version"]) > VERSION:
```

Comment:
We should probably check if `check50` was actually installed as a package, as via https://github.com/cs50/submit50/blob/develop/submit50.py#L32, since we wouldn't want to try updating if a user has installed/run from source?